lib/runner.js

```diff
@@ -5,6 +5,7 @@ var Promise = require('bluebird');
 var hasFlag = require('has-flag');
 var Test = require('./test');
 var send = require('./send');
+var objectAssign = require('object-assign');
```

Comment:
TODO: We probably should look in to replacing some of these polyfills with `core-js` equivalents. Hopefully cutting down on duplicate functionality.